lib/insights/src/lib/analytics-process/components/analytics-report-parameters.component.ts

```diff
@@ -187,11 +186,11 @@ export class AnalyticsReportParametersComponent implements OnInit, OnChanges, On
     }
 
     convertMomentDate(date: string) {
-        return moment(date, FORMAT_DATE_ACTIVITI, true).format(FORMAT_DATE_ACTIVITI) + 'T00:00:00.000Z';
+        return DateFnsUtils.formatDate(new Date(date), FORMAT_DATE_ACTIVITI) + 'T00:00:00.000Z'
```

Comment:
if you rebase, the `formatDate` already supports string input, so no need calling "new Date"

Reply:
This can be done from future PRs as discussed.